src/views/Users/ViewUser/index.jsx

```diff
@@ -86,7 +86,7 @@ function ViewUser({ isNewUser, ...props }) {
     },
   } = props;
   const classes = useStyles();
-  const [username, setUsername] = useState('');
+  const [username, setUsername] = useState(existingUsername);
```

Comment:
Thoughts on making this `const [username, setUsername] = useState(isNewUser ? '' : existingUsername);`?

Reply:
Probably a good idea, for some reason I thought existingUsername would be `''` in the new user case...